src/features/home/pages/ThematicHome.native.test.tsx

```diff
@@ -26,20 +26,21 @@ describe('ThematicHome', () => {
     thematicHeader: { title: 'HeaderTitle', subtitle: 'HeaderSubtitle' },
   })
 
-  it('should render correctly', () => {
+  it('should render correctly', async () => {
     // eslint-disable-next-line local-rules/no-react-query-provider-hoc
     const thematicHome = render(reactQueryProviderHOC(<ThematicHome />))
+    await screen.findByText('HeaderTitle')
     expect(thematicHome).toMatchSnapshot()
   })
 
-  it('should render default header when provided', () => {
+  it('should render default header when provided', async () => {
     // eslint-disable-next-line local-rules/no-react-query-provider-hoc
-    const { getByText } = render(reactQueryProviderHOC(<ThematicHome />))
-    expect(getByText('HeaderTitle')).toBeTruthy()
-    expect(getByText('HeaderSubtitle')).toBeTruthy()
+    render(reactQueryProviderHOC(<ThematicHome />))
+    expect(await screen.findByText('HeaderTitle')).toBeTruthy()
+    expect(screen.getByText('HeaderSubtitle')).toBeTruthy()
```

Comment:
Perso je pense que j'aurais fait ça : 
```suggestion
await waitFor(() => ({
    expect(screen.getByText('HeaderTitle')).toBeTruthy()
    expect(screen.getByText('HeaderSubtitle')).toBeTruthy()
})
```